cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamFileJanitorService.java

```diff
@@ -67,7 +68,7 @@ public void run() {
           janitor.cleanAll();
           LOG.debug("Completed stream file cleanup.");
         } catch (Throwable e) {
-          LOG.warn("Failed to cleanup stream file: {}", e.getMessage());
+          LOG.warn("Failed to cleanup stream files.", e);
```

Comment:
Does not look like error. We are not throwing the exception caught. Seems like intention is to catch all the errors while cleaning up and not propagate them right?